gammapy/stats/variability.py

```diff
@@ -2,30 +2,38 @@
 import numpy as np
 import scipy.stats as stats
 
-__all__ = ["compute_fvar", "compute_chisq"]
+__all__ = [
+    "compute_fvar",
+    "compute_chisq",
+]
 
 
-def compute_fvar(flux, flux_err):
+def compute_fvar(flux, flux_err, axis= 0):
     r"""Calculate the fractional excess variance.
 
-    This method accesses the the ``FLUX`` and ``FLUX_ERR`` columns
+    This method accesses the ``FLUX`` and ``FLUX_ERR`` columns
     from the lightcurve data.
 
     The fractional excess variance :math:`F_{var}`, an intrinsic
     variability estimator, is given by
 
     .. math::
-        F_{var} = \sqrt{\frac{S^{2} - \bar{\sigma^{2}}}{\bar{x}^{2}}}.
+        F_{var} = \sqrt{ \frac{S^{2} - \bar{ \sigma^{2}}}{ \bar{x}^{2}}}
 
     It is the excess variance after accounting for the measurement errors
     on the light curve :math:`\sigma`. :math:`S` is the variance.
 
+    It is important to note that the errors on the flux must be gaussian.
+
     Parameters
     ----------
     flux : `~astropy.units.Quantity`
         the measured fluxes
     flux_err : `~astropy.units.Quantity`
         the error on measured fluxes
+    axis : int, optional
+        Axis along which the excess variance is computed.
+        The default is to compute the value on axis 0, the default time axis in gammapy.
```

Comment:
```suggestion
        The default is to compute the value on axis 0.
```
I would remove the last part of the sentence. I am not sure this will always be the case.

Reply:
Thanks, edited